koku/koku/settings.py

```diff
@@ -17,11 +17,11 @@
 import os
 import sys
 from json import JSONDecodeError
-from zoneinfo import ZoneInfo
 
 from boto3.session import Session
 from botocore.exceptions import ClientError
 from corsheaders.defaults import default_headers
+from zoneinfo import ZoneInfo
```

Comment:
This can go somewhere in the imports.
```suggestion
from zoneinfo import ZoneInfo

try:
    from django_cprofile_middleware.middleware import ProfilerMiddleware
except ImportError:
    ProfilerMiddleware = None
```